src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,25 @@
 package mate.academy.hibernate.relations.model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@Entity
```

Comment:
Add `@Table` for each entity
it is needed to use naming convention for table name - plural form